ceiling_perception: #name space (same as launch file)
  ceiling_perception: # package name
    ros__parameters:
      enabled: true
      publish_tf: false
      log_odds_saturation: 10.0
      resolution: 0.05 #must be as same as SLAM resolution

      # simulation:
#      focal_x_: 254.91
#      focal_y_: 254.91
#      x_0_: 320.5
#      y_0_: 240.5
#      num_overhead_cameras: 4
#      overhead_topics: [ "/ceiling_segmentation/autonomous_robot/image_segmentation/image_1",
#                         "/ceiling_segmentation/autonomous_robot/image_segmentation/image_2",
#                         "/ceiling_segmentation/autonomous_robot/image_segmentation/image_3",
#                         "/ceiling_segmentation/autonomous_robot/image_segmentation/image_4"]
#      camera_poses: "[[-5.0, -5.0, 3.5], [0.5, -5.0, 3.5], [-5.0, -0.8, 3.5], [0.5, 0.8, 3.5]]"
      # real
      focal_x_: 381.52740478515625
      focal_y_: 381.52740478515625
      x_0_: 317.77301025390625
      y_0_: 250.42886352539062
      num_overhead_cameras: 1
      overhead_topics: [ "/ceiling_segmentation/autonomous_robot/image_segmentation/image_1"]
      camera_poses: "[[-5.0, -5.0, 3.5]]"
